graphql/types/ProductSearch.graphql

```diff
@@ -35,11 +37,23 @@ type ProductSearch {
   The possible values in this field are defined by each search engine.
   """
   searchState: String
-  banners: [SearchBanner] @deprecated(reason: "Use the `banners` query instead.")
+  banners: [SearchBanner]
+    @deprecated(reason: "Use the `banners` query instead.")
   """
   Returns the redirect URL set for the given query.
   """
   redirect: String
+  """
+  Options to ProductSearch.
+  """
+  options: ProductSearchOptions
+}
+
+type ProductSearchOptions {
+  """
+  Indicates if delivery promises is enabled.
+  """
+  deliveryPromisesEnabled: Boolean
```

Comment:
Same comment as  in https://github.com/vtex-apps/store-resources/pull/191#discussion_r2054760223